### ISSUE TITLE
Support for ZONE2 and ZONE3

### ISSUE THREAD

**Issue by docbobo** [docbobo/roon-extension-denon#2]
The current extension only supports the Main Zone for Volume Control. It should easily possible to refactor the code so that it also supports ZONE2 and ZONE3. However, things get a little bit more complicated when it comes to also supporting #1.

**Comment by docbobo** [comment]
Perhaps as user should be able to select the Zones in the settings?

**Comment by xkonni** [comment]
yes that would be awesome. having 2 rooms connected to my marantz nr1608 as different zones...
